lib/phi_attrs/phi_record.rb

```diff
@@ -525,6 +574,22 @@ def revoke_extended_phi!(relations = nil)
       @__phi_relations_extended.subtract(relations)
     end
 
+    # Adds a disallow PHI access to the stack for block syntax for all `extend`ed relations (or only those given)
+    def add_disallow_flag_to_extended_phi!(relations = nil)
+      relations ||= @__phi_relations_extended
+      relations.each do |relation|
+        relation.add_disallow_flag! if relation.present? && relation_klass(relation).included_modules.include?(PhiRecord)
+      end
+    end
+
+    # Adds a disallow PHI access to the stack for all for all `extend`ed relations (or only those given)
+    def remove_disallow_flag_revoke_extended_phi!(relations = nil)
```

Comment:
This should probably be `remove_disallow_flag_from_extended_phi!` ("from", not "revoke") to avoid confusion.